rotkehlchen/history/events.py

```diff
@@ -37,12 +37,14 @@
 # eth transactions
 # eth receipts
 # eth tx decoding
+# trades
+# asset movements
+# margin positions
 # ledger actions
-# external location trades -> len(EXTERNAL_LOCATION)
 # eth2
 # base history entries
 # Please, update this number each time a history query step is either added or removed
-NUM_HISTORY_QUERY_STEPS_EXCL_EXCHANGES = 6 + len(EXTERNAL_LOCATION)
+NUM_HISTORY_QUERY_STEPS_EXCL_EXCHANGES = 9
```

Comment:
this is quite dirty. We probably should think of a way to make this better, not have hard coded steps like this and comments. **Not an action item** for this PR since this has been there for a while. Just mentioning it.